core/server/middleware/request-id.js

```diff
@@ -0,0 +1,13 @@
+var uuid = require('uuid'),
+    _ = require('lodash'),
+    headerName = 'X-Request-Id',
+    idAttribute = 'requestId';
+
+module.exports = function requestId(req, res, next) {
+    req.userId = _.get(req, 'user.id', null);
```

Comment:
Plus: In my opinion, this line does not really fit into the concept of splitting these middleware 🤔  Maybe we keep this line in the log request middleware? It's optional anyway. Plus we could create a closure function in ignition which defines where the user can be found e.g.

`app.use(ignition.middlewares.logRequest({userIsIn: 'req.user'}))`